doc/rear-release-notes.txt

```diff
@@ -194,6 +194,17 @@ for you.
 
 The references pointing to fix #nr or issue #nr refer to our issues tracker.
 
+Version 2.5.rubrik (October 2019)
+
+Abstract
+
+Added support for Rubrik Cloud Data Management (CDM)
+
+New Features:
+
+  o Added support for Rubrik Cloud Data Management (CDM) as an integrated
+    backup application.
+
```

Comment:
@DamaniN 
your additions to doc/rear-release-notes.txt are not wrong
but we maintain the doc/rear-release-notes.txt source at a different place.
The one for the upcoming ReaR version 2.6 is there
https://github.com/rear/rear.github.com/blob/master/documentation/release-notes-2-6.md
where mainly @gdha and I add the changelog notes.
Finally its final content gets copied to  doc/rear-release-notes.txt directly before
ReaR 2.6 would be released so that your changes here would get overwritten.
So to keep this pull request clean I would think you may better
remove your additions to doc/rear-release-notes.txt here
and let me add the changelog notes to
https://github.com/rear/rear.github.com/blob/master/documentation/release-notes-2-6.md
as time permits after this pull request was merged.

Reply:
Restored the rear-release-notes.txt from the rear/rear/master